src/databricks/labs/ucx/install.py

```diff
@@ -401,6 +412,11 @@ def run_workflow(self, step: str):
             job_run = self._ws.jobs.get_run(job_run_waiter.run_id)
             raise self._infer_error_from_job_run(job_run) from err
 
+    def trigger_workflow(self, step: str):
```

Comment:
make this method private. dispaly a link to a job that has just been started to the user and do `webbrowser.open`.